guides/source/testing.md

```diff
@@ -479,7 +479,7 @@ number passed to the parallelize method.
 To enable parallelization add the following to your `test_helper.rb`:
 
 ```ruby
-class ActiveSupport::TestCase
+ActiveSupport.on_load'(:active_support_test_case) do
```

Comment:
Those are fine since they are in the `test_helper.rb` and it is after the `require_relative '../config/environment'`. Only gems and code that is loaded before that require that needs to use the `on_load`.